### DISCUSSION TITLE
Acknowledging Alert causes SOC to freeze

### DISCUSSION THREAD

**Discussion by ranguli** [Security-Onion-Solutions/securityonion#5464] [Unsupported Versions]
Hi all,

I initially thought I was experiencing issue #5221 described in the discussion #5201, however this appears to be different as the workaround fix using the _Include_ quick action does not seem to remedy the problem, nor have I found a combination of groupings/filters in my testing that eliminate this issue.

Here are some reproduction steps on a **distributed deployment** of **Security Onion 2.3.70**:
1. Acknowledge an alert in the SOC console, and the page will hang on the loading indicator.
<img width="1477" alt="Screen Shot 2021-09-10 at 11 03 01 AM" src="https://user-images.githubusercontent.com/5544782/132861999-54d72105-96fe-408f-a084-e330e9d6b091.png">
2. After a minute or so, two errors will appear at the same time, overlapping each other.
<img width="1476" alt="Screen Shot 2021-09-10 at 11 14 17 AM" src="https://user-images.githubusercontent.com/5544782/132863173-842226ee-9f68-4974-823e-a0c4d11a67ee.png">
3. On one occasion while trying to replicate this issue (and grab a screenshot in time before the error dissapeared) I got a different error, however I wasn't aware of anything I did differently to cause it.
<img width="1477" alt="Screen Shot 2021-09-10 at 11 11 40 AM" src="https://user-images.githubusercontent.com/5544782/132863289-f0a0d246-6d9b-4e2f-b39e-7ebc9b6b7abf.png">

I've also tried acknowledging single alerts but that does not change the outcome.

Thanks,
Joshua

**Comment by dougburks** [comment]
Take a look at the logs in `/opt/so/log/` to see if there are any additional clues. In particular, look at the SOC logs in `/opt/so/log/soc/` and the Elasticsearch logs in `/opt/so/log/elasticsearch/`:
https://docs.securityonion.net/en/2.3/elasticsearch.html#diagnostic-logging
